apps/root/src/pages/dca/create-position/components/landing/chart.tsx

```diff
@@ -0,0 +1,132 @@
+        fill="#978AA8"
+      />
+      <line x1="52" y1="108.741" x2="577" y2="108.741" stroke="#A399FF" strokeWidth="3" strokeDasharray="15 15" />
```

Comment:
This should be violet600 (themeMode-based)

Reply:
You sure? I exported the svg directly from figma and it exported the same one both for dark and light

Reply:
Maybe i'm too meticulous at this point 😅 But this is the dashed line, and in light mode is looking much brighter comparing it with the design. I find this particular vector to change based on the theme